docs/docs/troubleshooting/SUPPORT_BOUNDARIES.md

```diff
@@ -0,0 +1,53 @@
+# Support boundaries
+
+This page describes what is within our scope of support for Kubernetes deployments, and what isn't. 
+
+## Supported components
+### Helm charts
+
+Helm is a Kubernetes package manager. It allows us to provide generic YAML templates that you configure with the specific values for your environments.
+
+You are responsible for creating the components that are required by the product for your type of deployment, as describes in the [Prerequisites](../userguide/PREREQUISITES.md). You need to insert the values (hostnames, configuration values) into your specific `values.yaml` file that is used for installation. We provide documentation for different configuration options in the [Configuration guide](../userguide/CONFIGURATION.md) and the [Examples](../examples/EXAMPLES.md).
```

Comment:
The words:
`for your type of deployment, as describes`

should be:

`for your type of deployment, as described` 

replace `describes` with `described`